jmx-scraper/src/main/resources/hbase.yaml

```diff
@@ -0,0 +1,329 @@
+        metricAttribute:
+          state: const(flush)
+          region_server: *hostname
+      compactionQueueLength:
+        metric: *metric
+        unit: *unit
+        desc: *desc
+        metricAttribute:
+          state: const(compaction)
+          region_server: *hostname
+
+      updatesBlockedTime:
+        metric: blocked_update.time
+        type: gauge
+        unit: ms
+        desc: Amount of time updates have been blocked so the memstore can be flushed.
+
+      blockCacheMissCount:
+        metric: &metric block_cache.operation.count
+        type: &type gauge
+        unit: &unit "{operation}"
+        desc: &desc Number of block cache hits/misses.
+        metricAttribute:
+          state: const(miss)
+          region_server: *hostname
+      blockCacheHitCount:
+        metric: *metric
+        type: *type
+        unit: *unit
+        desc: *desc
+        metricAttribute:
+          state: const(hit)
+          region_server: *hostname
+
+      percentFilesLocal:
+        metric: files.local
+        type: gauge
+        unit: "%"
+        desc: Percent of store file data that can be read from the local.
+
+      slowDeleteCount:
+        metric: &metric operations.slow
+        unit: &unit "{operation}"
+        desc: &desc Number of operations that took over 1000ms to complete.
+        metricAttribute:
+          operation: const(delete)
+          region_server: &hostname beanattr(tag\.Hostname)
+      slowAppendCount:
+        metric: *metric
+        unit: *unit
+        desc: *desc
+        metricAttribute:
+          operation: const(append)
+          region_server: *hostname
+      slowGetCount:
+        metric: *metric
+        unit: *unit
+        desc: *desc
+        metricAttribute:
+          operation: const(get)
+          region_server: *hostname
+      slowPutCount:
+        metric: *metric
+        unit: *unit
+        desc: *desc
+        metricAttribute:
+          operation: const(put)
+          region_server: *hostname
+      slowIncrementCount:
+        metric: *metric
+        unit: *unit
+        desc: *desc
+        metricAttribute:
+          operation: const(increment)
+          region_server: *hostname
+
+  - bean: Hadoop:service=HBase,name=RegionServer,sub=Server
+    prefix: hbase.region_server.
+    type: gauge
+    unit: ms
+    metricAttribute:
+      region_server: *hostname
+    mapping:
+      Append_99th_percentile:
+        metric: operation.append.latency.p99
+        desc: Append operation 99th Percentile latency.
+      Append_max:
+        metric: operation.append.latency.max
+        desc: Append operation max latency.
+      Append_min:
+        metric: operation.append.latency.min
+        desc: Append operation minimum latency.
+      Append_mean:
+        metric: operation.append.latency.mean
+        desc: Append operation mean latency.
+      Append_median:
+        metric: operation.append.latency.median
+        desc: Append operation median latency.
```

Comment:
Not related to this PR, but I am pretty sure those are metrics [created with dropwizard](https://metrics.dropwizard.io/4.2.0/getting-started.html#reporting-via-jmx), we need to make sure those are mapped consistently for every supported system where they are used. The downside of those here is that there is no way for us to translate that easily to an otel histogram as we don't have access to the individual samples.